cmd/ipfs-cluster-service/main.go

```diff
@@ -30,6 +30,8 @@ const (
 	defaultLogLevel   = "info"
 )
 
+const stateCleanupPrompt = "The peer's state will be removed from the load path.  Existing pins may be lost.\nConfiguration(service.json) will be overwritten.  Continue? [y/n]:"
```

Comment:
There  should probably be a `stateCleanupPrompt` and a `configurationOverwritePrompt`.

For `inif -f` you can use `fmt.Sprintf("%s\n%s, stateCleanupPrompt, configurationOverwritePrompt)`.

The `Continue? [y/n]:` part can be added inside `yesNoPrompt()`.

The `stateCleanupPrompt`  can be re-used in `state cleanup`.